### ISSUE TITLE
Wrongly used dropout bug

### ISSUE THREAD

**Issue by qiuqiangkong** [tensorflow/tensorflow#15272]
Hi, 

    I guess it should use **tf.layers.dropout** instead of **tf.nn.dropout** here? Because in the inference stage, all the nodes should be used instead of dropout. I guess this is a bug. 

https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/examples/tutorials/mnist/mnist_deep.py#L92

    Many thanks! 

Best wishes, 

Qiuqiang

**Comment by qmick** [comment]
For `minst_deep.py`, the `keep_prob` argument of `tf.nn.dropout` is set to 1.0 to keep all nodes at test stage. It is not necessary to change to `tf.layers.dropout`.

**Comment by qiuqiangkong** [comment]
Many thanks! I see! I will close this issue!
